.sauce/config.yml

```
@@ -30,6 +30,16 @@ suites:
     testMatch: '**/*.js'
     params:
       browserName: "chromium"
+  - name: "Firefox in docker"
+    mode: docker
+    testMatch: '**/*.js'
+    params:
+      browserName: "firefox"
+  - name: "Webkit in docker"
+    mode: docker
+    testMatch: '**/*.js'
```

That needs to be regex now:

```suggestion
    testMatch: '.*.js'
```

(Error was not seen due to https://github.com/saucelabs/sauce-playwright-runner/pull/106)